incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -455,5 +456,108 @@ problem, and will generally require human intervention to resolve.
 
 > _Note: HNC v0.5 reported issues with objects as part of the non-standard
 > condition schema. These have been removed and replaced by standard Events in
-> HNC v0.6 since events are more standard, scalable and loggable.
+> HNC v0.6 since events are more standard, scalable and loggable._
+
+<a name="admin-labels">
+
+### Labels and annotations
+
+HNC both adds and respects certain annotations and labels on namespaces and
+objects. Typically, most users (or admins) will never need to care about these,
+but occasionally they may cause some odd changes in behaviour that you need to
+be aware of.
+
+#### app.kubernetes.io/managed-by
```

Comment:
Maybe indicate if it's label or annotation in the each sub-header?

Reply:
Done